charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/allow-to-alicloud-network.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress from pods labeled with 'networking.gardener.cloud/to-alicloud-networks=allowed'
+      to Alicloud CloudProvider's specific IPs. Currently, only consider 100.96.0.0/11.
+  name: allow-to-alicloud-networks
+  namespace: {{ .Release.Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      networking.gardener.cloud/to-alicloud-networks: allowed
+  egress:
+  - to:
+    - ipBlock:
+        cidr: 100.96.0.0/11
```

Comment:
Where did you get this CIDR from? Is this large range really taken by Alicloud for its services? What about existing shoots e.g. having a service CIDR like `100.104.0.0/13`? Are they now "suddenly" in Alicloud's IP range?

Reply:
Not suddenly. We probably made a wrong IP CIDR for default k8s service shoot. For Alicloud we need to avoid to use 100.**** CIDR in shoot. Here is my proof, though I couldn't find it in official doc from Alicloud website. 

- For SLB, ther prober IP addresses are 100.***. Here is the logs from kube-apiserver:
```
I0220 06:41:00.132147       1 log.go:172] http: TLS handshake error from 100.116.38.3:42351: read tcp 10.243.132.39:443->100.116.38.3:42351: read: connection reset by peer
I0220 06:41:00.264179       1 log.go:172] http: TLS handshake error from 100.97.48.1:25780: read tcp 10.243.132.39:443->100.97.48.1:25780: read: connection reset by peer
I0220 06:41:00.315692       1 log.go:172] http: TLS handshake error from 100.97.230.129:23638: read tcp 10.243.132.39:443->100.97.230.129:23638: read: connection reset by peer
I0220 06:41:00.361661       1 log.go:172] http: TLS handshake error from 100.97.231.1:11331: read tcp 10.243.132.39:443->100.97.231.1:11331: read: connection reset by peer
I0220 06:41:01.477995       1 log.go:172] http: TLS handshake error from 100.97.231.129:57459: read tcp 10.243.132.39:443->100.97.231.129:57459: read: connection reset by peer
I0220 06:41:02.147370       1 log.go:172] http: TLS handshake error from 100.116.38.3:44613: read tcp 10.243.132.39:443->100.116.38.3:44613: read: connection reset by peer
I0220 06:41:02.234262       1 log.go:172] http: TLS handshake error from 100.97.48.1:57409: read tcp 10.243.132.39:443->100.97.48.1:57409: read: connection reset by peer
I0220 06:41:02.247194       1 log.go:172] http: TLS handshake error from 100.97.230.129:14323: read tcp 10.243.132.39:443->100.97.230.129:14323: read: connection reset by peer
I0220 06:41:02.435394       1 log.go:172] http: TLS handshake error from 100.97.231.1:61973: read tcp 10.243.132.39:443->100.97.231.1:61973: read: connection reset by peer
```

- Meta service IP: 100.100.100.200
- VPC typed registry: for e.g. registry-vpc.cn-shanghai.aliyuncs.com, the ip is 100.103.7.56
- VPC typed API endpoint IP: 100.100.0.18

From the above oberservation, maybe we need to boarden it to 100.64.0.0/10. refer Carrier-grade NAT (RFC6598). At the same time, we need to find a suitable CIDRs for shoot in Alicloud.

Reply:
Thanks.